### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by pavlosoletskyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+This system allows users to add books to their libraries and search for them by title, author, or genre. The `LibraryService` class already contains skeleton methods you must fill out.
+
+Additionally, you are provided with two data classes: `Book` and `Author`.
+
+ - A `Book` contains a **title, list of authors, publication date, genre,** and **ISBN** fields.
+ - An `Author` data class has only one field, **name**.
 
-The `LibraryService` class already contains skeleton methods that you need to fill in. 
-Additionally, you are provided with two data classes: `Book` and `Author`. A `Book` has a title, a list of authors, a publication year, a genre, and an ISBN number. An `Author` has a name.
+##Instructions
```
Comment (alenatovstukha) [inline README.md:13]: space between ## and Instructions

File: README.md [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
 
-Your task includes:
+1. Create an `Author` class with a single field `name` of type `String`.
+
+2. Create a `Book` class with the fields described below. Please put them in the order as follows:
 
-1. Create an `Author` class with a single field `name` of type String
-2. Create a `Book` class with the following fields. Please put them in the order as described below: 
    - `title` of type `String`
```
Comment (alenatovstukha) [inline README.md:19]: - `title` of type `String`;

semicolon `;`

File: README.md [hunk 3]
```diff
@@ -1,21 +1,37 @@
...
+
    - `authors` of type `List<Author>`
```
Comment (alenatovstukha) [inline README.md:21]: `;`

File: README.md [hunk 4]
```diff
@@ -1,21 +1,37 @@
...
+
    - `publicationYear` of type `Int`
```
Comment (alenatovstukha) [inline README.md:23]: `;`

File: README.md [hunk 5]
```diff
@@ -1,21 +1,37 @@
...
+
    - `genre` of type `String`
```
Comment (alenatovstukha) [inline README.md:25]: `;`

File: README.md [hunk 6]
```diff
@@ -1,21 +1,37 @@
...
+
    - `isbn` of type `String`
```
Comment (alenatovstukha) [inline README.md:27]: `;`

File: README.md [hunk 7]
```diff
@@ -1,21 +1,37 @@
...
-3. Implementing the `addBook(book: Book)` method to add a new book to the library. HINT: use some collection here. Book duplicates are allowed.
-4. Implementing the `searchByTitle(title: String): List<Book>` method to return a list of books that contain the given title. The search should be case-insensitive.
-5. Implementing the `searchByAuthor(authorName: String): List<Book>` method to return a list of books written by authors whose names contain the given string. This search should also be case-insensitive.
-6. Implementing the `searchByGenre(genre: String): List<Book>` method to return a list of books that match the given genre exactly, with case-insensitivity.
+
+3. Implement the `addBook(book: Book)` method to add a new book to the library. 
+
+💡**HINT:** use some collection here. Book duplicates are allowed.
```
Comment (alenatovstukha) [inline README.md:31]: 💡 Use some collection here. Book duplicates are allowed.

**Review by alenatovstukha — CHANGES_REQUESTED** [review]

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**